src/test/scala/PacketAssembler_test/PacketDisassemblerTester.scala

```diff
@@ -74,18 +74,29 @@ class PacketDisAssemblerTest(c: PacketDisAssembler) extends PeekPokeTester(c) {
     }
     poke(c.io.in.data.valid, false.B)
     step(10)
+    poke(c.io.in.data.valid, true.B)
+
+    for (j <- 0 to 7) {
```

Comment:
Maybe make the pre-preamble an optional part of the tester and you can test both?

Reply:
I can test pre-preamble in half of the 100 random packets

Reply:
Not sure I followed what you mean

Reply:
in packet disassembler test there is a for loop that generates 100 random packets and passes them through the module - i can test pre-preamble in 50 packets and not test it in the other 50

Reply:
Is that currently being done?

Reply:
it is now